flask/app.py

```diff
@@ -1991,6 +1991,9 @@ def wsgi_app(self, environ, start_response):
             except Exception as e:
                 error = e
                 response = self.handle_exception(e)
+            except:
```

Comment:
Instead of `except:` and using `sys.exc_info` you could use `except BaseException as e`